src/components/MessageBanner.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { useTheme } from '@react-navigation/native';
+
+export const MessageBanner = ({
+  message = "There's no movie added to My List yet",
+  customStyles,
+}) => {
+  const { colors } = useTheme();
+
+  return (
+    <View style={[customStyles.messageContainer]}>
```

Comment:
this does not need to be an array:
`<View style={customStyles.messageContainer}>`